testing/content/files/en-us/web/bar/index.html

```diff
@@ -20,6 +20,8 @@
 	<li>{{jsxref("Stern_mode")}}</li>
 	<li>{{jsxref("Strict_mode")}}</li>
 	<li>{{jsxref("Flag")}}</li>
+	<li>{{ jsxref('Flag') }}</li>
+	<li>{{JSXref("Flag")}}</li>
```

Comment:
Perhaps it's pointless (due to my ignorance) but would it be helpful to assert that if two perfectly identical macros appear too. E.g. `{{jsxref("Flag")}}` and `{{jsxref("Flag")}}`.
Because that's actually a pretty interesting challenge later with regards to the `--fix-flaws`, which we're working on, which uses string replace on the source and string replace only works when the case, quotation mark, whitespace and all is perfectly identical.

Reply:
Good idea. I'll add that.